src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -768,6 +768,14 @@ To add a post-script to a task, use:
 #%post_script(name=TASK_NAME, language=SCRIPT_LANGUAGE, [path=./POST_SCRIPT_FILE.py])
 ```
 
+==== Deleting a task
```

Comment:
We could choose either gerund or infinitive verb tense, but use the same all along the items for consistency (i.e. Add, Delete, Create, or Adding, Deleting, Creating, ...). Just a suggestion.

Reply:
You're right!